projects/js-packages/charts/tools/webpack.common.cjs

```diff
@@ -0,0 +1,83 @@
+const { CleanWebpackPlugin } = require( 'clean-webpack-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+
+// List of components to build individually
+const components = [
+	'components/bar-chart',
+	'components/line-chart',
+	'components/pie-chart',
+	'components/pie-semi-circle-chart',
+	'components/tooltip',
+	'components/legend',
+	'components/grid-control',
+	'providers/theme',
+];
+
+// Common configuration for both ESM and CommonJS builds
+const getCommonConfig = isESM => ( {
+	module: {
+		rules: [
+			{
+				test: /\.(ts|tsx)$/,
+				use: [
+					{
+						loader: 'babel-loader',
+						options: {
+							presets: [ '@babel/preset-env', '@babel/preset-react', '@babel/preset-typescript' ],
+							plugins: [ [ '@babel/plugin-transform-runtime', { useESModules: isESM } ] ],
+						},
+					},
+				],
+				exclude: /node_modules/,
+			},
+			{
+				test: /\.(scss|css)$/,
+				use: [
+					MiniCssExtractPlugin.loader,
+					{
+						loader: 'css-loader',
+						options: {
+							modules: {
+								localIdentName: '[name]__[local]__[hash:base64:5]',
```

Comment:
Would this apply only to files ending with `.module.css/scss`? I think it would be best to avoid making modules from regular CSS files, in case we want to include global files or some contributors need to opt out of the modules.

Reply:
Good catch. The code is not in use so remove it 👍

Reply:
> The [css-loader](https://github.com/webpack/css-loader) has CSS Modules built-in. Simply activate it by using the ?modules flag. We maintain an example project using this at [css-modules/webpack-demo](https://css-modules.github.io/webpack-demo/). - [Ref](https://github.com/css-modules/css-modules/blob/master/docs/get-started.md)

It's [enabled by default ](https://github.com/webpack-contrib/css-loader?tab=readme-ov-file#modules)and only files with .module.  and a couple of others will be through the CSS modules handling

> undefined - enable CSS modules for all files matching /\.module\.\w+$/i.test(filename) and /\.icss\.\w+$/i.test(filename) regexp.